lib/ui5Framework/maven/Installer.js

```diff
@@ -0,0 +1,497 @@
+import path from "node:path";
+import {mkdirp} from "../../utils/fs.js";
+import fs from "graceful-fs";
+import _StreamZip from "node-stream-zip";
+const StreamZip = _StreamZip.async;
+import {promisify} from "node:util";
+import Registry from "./Registry.js";
+import AbstractInstaller from "../AbstractInstaller.js";
+import _rimraf from "rimraf";
+const rimraf = promisify(_rimraf);
```

Comment:
rimraf returns promises instead of callbacks with version 4, so this promisify is not needed.

Reply:
Thanks!